[2025-03-11T05:36:20.113+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.fetch_purchases manual__2025-03-11T05:35:48+00:00 [queued]>
[2025-03-11T05:36:20.118+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.fetch_purchases manual__2025-03-11T05:35:48+00:00 [queued]>
[2025-03-11T05:36:20.118+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2025-03-11T05:36:20.124+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): fetch_purchases> on 2025-03-11 05:35:48+00:00
[2025-03-11T05:36:20.126+0000] {standard_task_runner.py:57} INFO - Started process 713 to run task
[2025-03-11T05:36:20.128+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ingest_bronze_layer_dag', 'fetch_purchases', 'manual__2025-03-11T05:35:48+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/ingesting_pipeline.py', '--cfg-path', '/tmp/tmpyt4tjsy5']
[2025-03-11T05:36:20.130+0000] {standard_task_runner.py:85} INFO - Job 63: Subtask fetch_purchases
[2025-03-11T05:36:20.151+0000] {task_command.py:416} INFO - Running <TaskInstance: ingest_bronze_layer_dag.fetch_purchases manual__2025-03-11T05:35:48+00:00 [running]> on host 41ccd9d1769c
[2025-03-11T05:36:20.545+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andrew Tran' AIRFLOW_CTX_DAG_ID='ingest_bronze_layer_dag' AIRFLOW_CTX_TASK_ID='fetch_purchases' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T05:35:48+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T05:35:48+00:00'
[2025-03-11T05:36:20.551+0000] {base.py:73} INFO - Using connection ID 'postgres_source_db' for task execution.
[2025-03-11T05:36:20.559+0000] {logging_mixin.py:154} INFO - Incremental time: 1900-01-01 00:00:00
[2025-03-11T05:36:20.564+0000] {warnings.py:109} WARNING - /opt/***/dags/ingesting_pipeline.py:61: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)

[2025-03-11T05:36:20.597+0000] {logging_mixin.py:154} INFO - DataFrame:             timestamp  ...                     extracted_at
0 2022-04-01 22:53:05  ... 2025-03-11 05:36:20.565023+00:00
1 2022-04-01 22:53:05  ... 2025-03-11 05:36:20.565023+00:00
2 2022-04-01 19:03:22  ... 2025-03-11 05:36:20.565023+00:00
3 2022-04-03 17:17:38  ... 2025-03-11 05:36:20.565023+00:00
4 2022-04-02 11:53:15  ... 2025-03-11 05:36:20.565023+00:00

[5 rows x 7 columns]
[2025-03-11T05:36:20.602+0000] {base.py:73} INFO - Using connection ID 'postgres_warehouse_db' for task execution.
[2025-03-11T05:36:20.603+0000] {logging_mixin.py:154} INFO - URI: postgresql://***:***@postgres:5432/warehouse_db?__extra__=%7B%7D
[2025-03-11T05:36:20.607+0000] {base.py:73} INFO - Using connection ID 'postgres_warehouse_db' for task execution.
[2025-03-11T05:36:20.635+0000] {python.py:194} INFO - Done. Returned value was: Inserted 76 rows into bronze_layer.bronze_purchases
[2025-03-11T05:36:20.692+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ingest_bronze_layer_dag, task_id=fetch_purchases, execution_date=20250311T053548, start_date=20250311T053620, end_date=20250311T053620
[2025-03-11T05:36:20.723+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-11T05:36:20.735+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
